src/js/api/brand.js

```diff
@@ -165,6 +165,48 @@ ripe.Ripe.prototype.getInitials3dDisplacementTextureUrl = function(options) {
     return null;
 };
 
+/**
+ * Returns the URL for the metallic texture for the initials mesh.
+ *
+ * @param {Object} options A map with options, such as:
+ *  - 'brand' - The brand of the model.
+ *  - 'model' - The name of the model.
+ *  - 'version' - The version of the build, defaults to latest.
+ * @returns {String} The URL of the initials metallic texture.
+ */
+ripe.Ripe.prototype.getInitials3dMetallicTextureUrl = function(options) {
+    options = this._getMeshOptions(options);
+    return null;
+};
+
+/**
+ * Returns the URL for the normal texture for the initials mesh.
+ *
+ * @param {Object} options A map with options, such as:
+ *  - 'brand' - The brand of the model.
+ *  - 'model' - The name of the model.
+ *  - 'version' - The version of the build, defaults to latest.
+ * @returns {String} The URL of the initials normal texture.
+ */
+ripe.Ripe.prototype.getInitials3dNormalTextureUrl = function(options) {
+    options = this._getMeshOptions(options);
+    return null;

```

Comment:
just checking... this is a TODO for the future but for now we're returning null, right?

Reply:
Yes